application/src/main/java/org/opentripplanner/routing/linking/mapping/LinkingContextRequestMapper.java

```diff
@@ -0,0 +1,25 @@
+package org.opentripplanner.routing.linking.mapping;
+
+import org.opentripplanner.routing.algorithm.raptoradapter.router.FilterTransitWhenDirectModeIsEmpty;
```

Comment:
Should we move this somewhere else? It's not used in raptoradapter and now there is a circular dependency between linking and raptoradapter. I'm not 100% sure should this even be used here or not. We need to resolve the actual direct mode for the linking somewhere.

Reply:
The `raptoradapter` contains more than just a facade for Raptor, so I this case there is no better place for this. This should go into a set of internal commands for the RoutingWorker, but no such place exist. There are more of these, so moving one of them will only make things worse. I think we should wait with this.